lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
So to the @joshcooper point from earlier, where he wants to specify a manifest like:

```puppet
scheduled_task { 'something': 
  command => '...'.
  minutes_interval => 30,
}
```

I created what he wants using schtasks:

```
schtasks.exe /create /sc minute /tn foo /tr C:\Windows\System32\notepad.exe /v1
```

And then used Puppet to describe it, and we see that he is spot on about `'minutes_duration' => '1440'`

```puppet
scheduled_task { 'foo':
  ensure      => 'present',
  command     => 'C:\Windows\System32\notepad.exe',
  enabled     => 'true',
  trigger     => [{'enabled' => 'true', 'every' => '1', 'index' => '0', 'minutes_duration' => '1440', 'minutes_interval' => '1', 'schedule' => 'daily', 'start_date' => '2015-4-2', 'start_time' => '11:06'}],
  user        => 'VAGRANT-2008R2\Administrator',
  working_dir => 'C:\Windows\System32',
}
```

So the question is - do we want to get a little fancier about how we set `minutes_duration` automatically when other settings are not present?